src/app/components/audios/audio-view/audio-view.component.ts

```diff
@@ -147,11 +147,13 @@ export class AudioViewComponent implements OnDestroy {
   }
 
   onDownloadClicked() {
-    this.audioRequestService.downloadAudio(this.route.snapshot.params.requestId).subscribe({
-      next: (blob: Blob) => {
-        this.saveAs(blob, `${this.audioRequest?.caseNumber.toString()}.zip`);
-      },
-    });
+    this.audioRequestService
+      .downloadAudio(this.route.snapshot.params.requestId, this.audioRequest.requestType)
+      .subscribe({
+        next: (blob: Blob) => {
+          this.saveAs(blob, `${this.audioRequest?.output_filename}.${this.audioRequest?.output_format}`);
```

Comment:
I think we need to confirm if the filename includes the extension, there was some discussion about it.

Reply:
See https://tools.hmcts.net/jira/browse/DMP-1417, looks like this should just use `output_filename` for the name and can use, if required, the format as the mime-type.